README.md

```diff
@@ -1,2 +1,50 @@
-# Solana Improvement Documents
-Solana IMprovement Documents (SIMD) describe proposed and accepted changes to the Solana protocol.
+# Solana Improvement Documents (SIMDs)
+
+The goal of the SIMD project is to standardize and provide high-quality
+documentation for Solana and its ecosystem. This repository tracks past and
+ongoing improvements to Solana in the form of Solana Improvement Documents
+(SIMDs).
+[SIMD-0001](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0001-simd-process.md)
+governs the SIMD process.
+
+## SIMD Types
+
+SIMDs can be divided into the following categories:
+
+- **Standard SIMDs**:
+  Describe changes that affect most or all Solana implementations, such as:
+  - **Core**:
+    Changes affecting consensus or substantial changes to the validator.
+  - **Networking**:
+    Changes or substantial improvements to network protocol specifications.
+  - **Interfaces**:
+    Breaking changes around the client JSON RPC API specifications and standards.
```

Comment:
i don't believe json rpc is under simd (nor should it be). svm et. al probably a better example for an interfaces bullet point

Reply:
I had a simiiliar thought. I would like to address this an a separate change further down the line. This mirrors SIMD-0001 for now